extension/controller/translation/translationWorker.js

```diff
@@ -224,10 +232,15 @@ class TranslationHelper {
               return;
             }
             this.engineState = this.ENGINE_STATE.LOADED;
+            let notificationMessage = "Automatic Translation enabled";
+            if (!isReversedModelLoaded) {
+                notificationMessage += "; Translation of forms disabled (reverse translation is not supported for this language)"
```

Comment:
Please, replace this by: `notificationMessage += ". Translation of forms is not supported for this language."`

Reply:
done